### PR TITLE
Validate arguments

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: lib/specializer.js [hunk 1]
```diff
@@ -42,6 +43,7 @@ var resolver = require('shortstop').create(),
 // for same template in the right order.
 
 function templateResolve(config) {
+    validate("O", arguments);
```
Comment (grawk) [inline lib/specializer.js]: ' instead of " to match surrounding style

Comment (aredridel) [inline lib/specializer.js]: Fixed

**Comment by grawk** [comment]
:+1:

**Comment by pvenkatakrishnan** [comment]
what a great idea for a module. :+1:

**Comment by aredridel** [comment]
Yeah! I love `aproba`. The "what it won't ever do" is great.

**Pull request merged**